style-guide/blog-guidelines.rst

```diff
@@ -0,0 +1,181 @@
+.. _blog-guidelines:
+
+===============
+Blog guidelines
+===============
+
+Developers often ask an Information Developer to edit a blog that they
+have written and placed in the
+https://github.com/rackerlabs/docs-developer-blog repository. Blogs in this
+repository are published to the Rackspace Developer Blog site at
+https://developer.rackspace.com/blog/.
+
+When you write a blog, use the writing guidelines in this Style Guide as you
+would with other documentation that is curated by the Information Development
+team. Start with the information in the :ref:`quickstart`.
+
+This topic provides additional guidelines specific to writing blogs for the
+Rackspace Developer Blog site.
+
+
+
+Things to consider before writing a blog
+----------------------------------------
+
+Before you start writing, consider the following questions:
+
+- What is the goal of this developer blog? Describe it in two sentences or
+  less.
+
+  - Is it to be helpful and answer a question or offer a How-To?
+  - Is it to provide “thought leadership” or show our expertise in an area?
+    If so, describe what you’d like to convey.
+  - Is it something else altogether? If so, what? It’s possible that your
+    idea might be better presented as an article in the How-To or as a
+    technical or marketing white paper.
+
+- What is the significance of the content of the blog? Is it a new offering?
+  How is it helpful to existing or potential customers? Does it frame our
+  expertise is a new way?
+- Explain how the content of the blog aligns with, supports, or strengthens
+  Rackspace’s current business priorities and strategy.
+- Who is the target audience for this blog?
+- What specific customer problems or pain points does the blog address?
+  Provide names of customers willing to be references in support of the post.
+- Does the blogs describe something that Rackspace does better than the
+  competition or that no other competitor offers? Does the _blog provide
+  quantifiable and defensible information (such as performance metrics)
+  showing the Rackspace edge over the competition? If yes, explain.
+
+
+Blog writing suggestions
+------------------------
+
+When you write your content, we recommend the following rules of thumb:
+
+-  Length: Rackspace blog posts generally run between 500-1,000 words, but let
+   the content be your guide. Take the space you need to tell your story, but
+   don’t pad it unnecessarily. Going long is okay, too, as long as it’s worth
+   reading.
+
+-  Tools: We recommend using a word processor, like MicroSoft Word, or a text
```

Comment:
I don't think "Microsoft" is supposed to have a capital "S" in it

Reply:
fixed

Reply:
Do we even recommend a word processor like Microsoft?

Reply:
fixed